.travis.yml

```diff
@@ -20,3 +19,11 @@ script:
   - "tox"
 after_success:
   coveralls
+deploy:
+  provider: pypi
+  user: edx
+  distributions: sdist bdist_wheel
+  skip_upload_docs: true
+  password:
+    secure: the-encrypted-password
```

Comment:
@efischer19 Does this page answer your question?:
https://openedx.atlassian.net/wiki/display/OpenOPS/Publishing+a+Package+to+PyPI+using+Travis